### PR TITLE
[DEV-17228] Add markup and content for premium plugins promos [4.x]

### PR DESCRIPTION
**Pull request by michalkleiner**
### Description:

4.x version of #21402

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/ProfessionalServices/templates/pluginAdvertising.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                    {% endfor %}
+                </ul>
+            </div>
+        {% endif %}
+    </div>
+    <div class="promo-actions">
+        <a class="btn" target="_blank" rel="noreferrer noopener" href="{{ plugin.shop.url|default('')|e('html_attr') }}">
+            {{ 'ProfessionalServices_CTAStartFreeTrial'|translate }}
+        </a>
+        <a class="learn-more" href="#" vue-directive="Marketplace.PluginName" vue-directive-value="{{ { pluginName: plugin.name }|json_encode }}" aria-label="{{ 'ProfessionalServices_CTALearnMore'|translate(plugin.displayName) }}">
```
Comment (michalkleiner) [inline plugins/ProfessionalServices/templates/pluginAdvertising.twig:23]: Just highlighting here that this is the only difference compared to the code for 5.x where the directive is called [`CorePluginsAdmin.PluginName`](https://github.com/matomo-org/matomo/pull/21402/files#diff-31afc88a22c4f772cefd35f5ab48267d2352e691864b10d6ae6e612bedd99dbcR23) and here for 4.x it's called `Marketplace.PluginName`.

**Review by caddoo — APPROVED** [review]

**Comment by michalkleiner** [comment]
Port of #21402 to 4.x-dev.

**Review by bx80 — APPROVED** [review]
Looks good to me :+1:

Any particular reason why Crash Analytics, A/B tests, Forms, Funnels, and Media Analytics use 'overview' as the stub page menu item, but Session recording and Heatmaps use 'Manage'?

**Comment by caddoo** [comment]
> Looks good to me 👍
> 
> Any particular reason why Crash Analytics, A/B tests, Forms, Funnels, and Media Analytics use 'overview' as the stub page menu item, but Session recording and Heatmaps use 'Manage'?

I can help here, it's just matching those main menu item names that appear when you use the premium plugins.

**Pull request merged**